### DISCUSSION TITLE
How to use this library in actual XD files?

### DISCUSSION THREAD

**Discussion by dfrechette** [pklaschka/xd-localization-helper#270] [Q&A]
Hi, this is likely a dumb question, but I don't know. How do you go about using this library in actual XD files? I can't find it in XD's plug-in menu.

**Comment by pklaschka** [comment]
Hi Daniel,

Thanks for asking!

`xd-localization-helper` isn't, in itself, a plugin for Adobe XD. Instead, it's a tool that helps plugins localize their UI (e.g., in the plugin panel or modal dialogs shown by a plugin). For example, I use it in my "Lorem Ipsum" plugin to localize its options dialog.

If you're looking for a plugin to localize your designs, here are a few suggestions (depending on your exact needs, different plugins might be ideal):

- [Crowdin XD Plugin](https://blog.crowdin.com/2020/08/10/crowdin-for-adobe-xd-translate-static-visuals-and-localize-product-ui/) - an integration of XD designs with the Crowdin platform
- [Docs2Design](http://emsoftware.com/xdplugins/docs-2-design/) - a plugin that allows to connect documents (with text, e.g., spreadsheets) to Adobe XD in multiple variants (e.g., multiple languages)

(there are a few more plugins which I can't remember right now)

I hope this helps, but if you have any additional questions / my answer doesn't make sense to you, please feel free to ask again!

Thanks,
Pablo
